docs/react-samples/src/App.tsx

```diff
@@ -1,26 +1,20 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import store from '@webex/cc-store'
 import {StationLogin} from '@webex/cc-station-login';
 import {UserState} from '@webex/cc-user-state';
 
 function App() {
-  const token = '';
+  const [isSdkReady, setIsSdkReady] = useState(false);
+  const [accessToken, setAccessToken] = useState("");
 
   const webexConfig = {
       fedramp: false,
       logger: {
-      level: 'log'  // TODO: We will add more logging levels later and set the righ levels
+        level: 'log'
       },
+      credentials: undefined,
```

Comment:
Why do we need to add credentials here ?

Reply:
I added it for testing the credentials config and forgot to remove it. Good catch. Thank you.